android/java/org/chromium/chrome/browser/shields/BraveShieldsHandler.java

```diff
@@ -98,13 +106,25 @@ public BlockersInfo() {
             mHTTPSUpgrades = 0;
             mScriptsBlocked = 0;
             mFingerprintsBlocked = 0;
+            mBlockerNames = new ArrayList<String>();
         }
 
         public int mAdsBlocked;
         public int mTrackersBlocked;
         public int mHTTPSUpgrades;
         public int mScriptsBlocked;
         public int mFingerprintsBlocked;
+        public ArrayList<String> mBlockerNames;
+    }
+
+    private static class DisconnectEntity {
```

Comment:
What is the meaning of disconnect entity? I didn't find the description in the docs and in the issue.

Reply:
It is to identify the ads/tracker to the big tech company. we are using the same json file in the ios as well.

Reply:
thanks